pages/linux/n.md

```diff
@@ -0,0 +1,23 @@
+# n
+
+> Tool to manage multiple node versions.
+
+- Install a given version of node. If the version is already installed, it will be activated:
+
+`n {{version}}`
+
+- Display installed versions and activate any version:
```

Comment:
"activate any version" seems a bit confusing. Does it mean that it activates the latest version automatically if one hasn't been activated yet?